# Battery status: voltage, percent, time, and plugged in indication

float32     voltage         # if applicable
float32     percent         # if applicable
float32     time            # if applicable
bool        plugged_in
